sickbeard/webserve.py

```diff
@@ -443,15 +443,6 @@ def setHistoryLayout(self, layout):
 
         return self.redirect("/history/")
 
-    def setHistoryLimit(self, history_limit):
```

Comment:
Yes, because the logic is done directly in `/history` handling.
I don't understand how that could affect saving in `config.ini` nor how it was done before. This method only sets a value, it does not save anything.